README.md

```diff
@@ -0,0 +1,42 @@
+# cross_modality_query
+
+Server-side code for HuBMAP Cells API.
+[Python](https://github.com/hubmapconsortium/hubmap-api-py-client) and 
+[Javascript](https://github.com/hubmapconsortium/hubmap-api-js-client) clients available.
+
+## Usage
+
+This is only a high-level outline of the API:
+See the [Python client examples](https://github.com/hubmapconsortium/hubmap-api-py-client/tree/main/examples) for details.
+
+The API supports a small number of basic operations which can be combined to construct more complex queries.
+The API runs at a **`base_url`**: Currently `https://cells.dev.hubmapconsortium.org/api` is available.
+
+Issuing a `POST` to `{base_url}/{output_type}/`, with query parameters in the body,
+will return a **query handle** representing `output_type` entities.
+`output_type` is currently limited to `cell`, `organ`, `gene`, and `cluster`.
+(The Python and Javascript interfaces provide a **query** abstraction, so you don't need to deal directly with the handle.)
+
+Issuing a `POST` to `{base_url}/{operation}/` (where `operation` is `union`, `intersection`, or `difference`),
+with query handles provided as `key_one` and `key_two` in the body, will return a new query handle,
+representing the result of the operation.
+
+Three endpoints are provided for getting more information, given a query handle:
+- `{base_url}/count/` will return the number of matching entities.
+- `{base_url}/evaluation/` will return a limited set of fields from the entities.
+- `{base_url}/detailevaluation/` can sort the results, and may return more fields, but may be a slower operation.
```

Comment:
@SFD5311 @mccalluc I am not very fond of the uncertainty in this statement. What does "may return more fields" mean?

Reply:
@ngehlenborg The detail evaluation, in addition to the fields returned by the basic list evaluation, returns additional values requested by the user, i.e. expression of a specified gene for a particular cell.  If no additional values are requested, the response has the same fields as the basic list evaluation.

Reply:
Ok. A better description of that endpoint would be "will return a both pre-defined and user-defined fields from the entities selected by the query in a user-defined or random order."

Not sure if "random" order is correct.

Any comments about retrieval performance (quickly, slower, etc.) should maybe follow in a separate paragraph below.

Reply:
I've push a fix which moves performance details to separate paragraph. I've described the ordering as "arbitrary" rather than "random".